amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:10.6-slim
```

Comment:
What is the advantage of these changes? Does it make the container smaller or larger?

Reply:
The `FROM` is smaller and ubuntu:20.04 is newer which supports more up to date packages. It's a continuation of the the ubuntu branch.